### PR TITLE
Add CW21 abstract and invite volunteers to co-lead this

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* The Turing Way abstract for CW21 has been accepted

### List of changes proposed in this PR (pull-request)

* Add abstract and invite interested folks to co-facilitate this

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?
- [ ] Are you interested in joining the event?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by EKaroune** [comment]
Looks good! Happy to help in any way I can. 😺

File: conferences/abstracts/2021-03-CW21.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Good Practices for Designing Software Development Projects (The Turing Way)
+
+### Mini-workshop or demo session abstract
+Effective methods for project design are crucial to the success of a research project. 
+Particularly in software development, design principles can lead to better code, maintainability, and extendability. 
+Project design encompasses a variety of aspects, starting from defining the purpose, main research questions, expected users and target audience, available resources and skills required in the project. 
+It is also important to explore the possible outcomes, plans to address expected challenges, ensure diversity of stakeholders and reduce possible barriers to participation.
+
+The Turing Way is an open-source, community-led book project that aims to bring together diverse contributors and collaborators to share resources and practices that make data science reproducible, ethical and inclusive. 
+The project is developed and maintained on an online project repository (https://github.com/alan-turing-institute/the-turing-way) and invites contributions to its 5 guides, including the guide for project design.
```
Comment (EKaroune) [inline conferences/abstracts/2021-03-CW21.md:23]: better to write 5 in words - five.

Also capitalise guide title in Guide for Project Design

**Review by EKaroune — APPROVED** [review]
all looks good!

**Comment by malvikasharan** [comment]
Thank you @EKaroune. Let's discuss this next week to plan the session.

File: conferences/abstracts/2021-03-CW21.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+* Website: https://www.software.ac.uk/cw21
+* Location: Online
+* Date: 30 March - 01 April 2021
+
+## Authors
+Malvika Sharan
+
+## Session collaborators
+TBA: SSI Fellows and volunteers from The Turing Way community
```
Comment (malvikasharan) [inline conferences/abstracts/2021-03-CW21.md:11]:
```suggestion
Emma Karoune, More SSI Fellows and volunteers from The Turing Way community
```

**Comment by BatoolMM** [comment]
I'm also happy to help in anyway I can ❄️

**Comment by malvikasharan** [comment]
@BatoolMM That would be great! I will ping you both on Slack next week to set the planning and discussion call.

**Pull request merged**